src/multicall.ts

```diff
@@ -539,15 +539,15 @@ export class Multicall {
         return '0x21681750D7ddCB8d1240eD47338dC984f94AF2aC';
       case Networks.arbitrum:
         return '0x80C7DD17B01855a6D2347444a0FCC36136a314de';
-      case Networks.avalauncheFuji:
+      case Networks.avalancheFuji:
```

Comment:
You need to add the case statement for the 2 networks you added!

Reply:
Added in d3de741

(I didn't save that file... Whoops.)